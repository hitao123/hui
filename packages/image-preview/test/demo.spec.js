describe('ImagePreview component', () => {
  it('should show ImagePreview', () => {

  });
});
