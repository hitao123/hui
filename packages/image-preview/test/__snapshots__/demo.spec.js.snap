// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div><button class="h-button h-button--default h-button--normal">
      <!----> <span class="h-button__text">图片预览</span></button></div>
  <div><button class="h-button h-button--default h-button--normal">
      <!----> <span class="h-button__text"></span></button></div>
</div>
`;
